src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -197,6 +197,15 @@ QtObject:
   QtProperty[QVariant] totalFees:
     read = getTotalFees
 
+  proc getMaxTotalFees*(self: ActivityEntry): QVariant {.slot.} =
+    if self.transaction == nil:
+      error "getMaxTotalFees: ActivityEntry is not an transaction.Item"
+      return newQVariant(newCurrencyAmount())
+    return newQVariant(self.transaction[].getMaxTotalFees())
+
+  QtProperty[QVariant] maxTotalFees:
+    read = getMaxTotalFees
```

Comment:
A getter w/o a notifier signal raises a red flag here :)

Reply:
Alright, I see, I was just triggered by the missing `CONSTANT` specifier which doesn't exist in NIM :)